helm/basex-validator/templates/service.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "basex-validator.fullname" . }}
+  annotations:
+    external-dns.alpha.kubernetes.io/hostname: {{ include "basex-validator.fullname" . }}.elifesciences.org
+    service.beta.kubernetes.io/aws-load-balancer-ssl-cert: {{ .Values.clientCertificate }}
+    service.beta.kubernetes.io/aws-load-balancer-backend-protocol: http
+    service.beta.kubernetes.io/aws-load-balancer-ssl-ports: "443"
+  labels:
+    app.kubernetes.io/name: {{ include "basex-validator.name" . }}
+    helm.sh/chart: {{ include "basex-validator.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  type: LoadBalancer
+  ports:
+    - port: {{ .Values.service.port }}
```

Comment:
supporting HTTP-only? Could this `port` be removed without losing functionality?